core/block_svg.js

```diff
@@ -1640,6 +1640,11 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
     }
   }
   Blockly.utils.dom.stopTextWidthCache();
+
+  var cursor = this.workspace.getCursor();
+  if (this.pathObject.cursorSvg_ && cursor.getDrawer()) {
```

Comment:
You still don't need the check for the drawer, because the cursor is doing that check inside `draw`